app/frontend/styles/_application-tabs.scss

```diff
@@ -0,0 +1,16 @@
+.app-tabs {
+  margin-top: govuk-spacing(8);
+
+  &__tab {
+    font-weight: bold !important;
+    text-align: left !important;
+    text-decoration: none !important;
+  }
+
+  &__list-item--selected {
+    background-color: govuk-colour("white") !important;
+    border-top: govuk-spacing(1) solid $govuk-link-colour !important;
+    color: $govuk-text-colour !important;
+    padding-top: govuk-spacing(2) !important;
+  }
```

Comment:
@tvararu what's our opinion on SCSS shortcuts like this? I'd prefer explicit & non-nested CSS.

Reply:
I have no strong opinions either way - but this is the style me and @tvararu used for in `_application-card.scss` for the `ApplicationCardComponent`.

Reply:
Yeah, that's actually where my preference comes from! I was fiddling with the card CSS yesterday and I searched for one of the classes in my editor. I was really confused why it didn't show up.

Reply:
haha, right yeah I can see that being an issue! I quite like the succinct / DRYness of being able to nest things though.

Is this something that you think I should address in this PR, or something we can take note of and discuss as a group? Similar to how we were speaking about JS styles / patterns - it might be good to have a conversation around SCSS as well. What do you think?

Reply:
I quite like this per-file as it follows the BEM hierarchy & usually grepping for `__thing` gets what you need. Maybe one for dev retro?